guides/manual-installation.md

```diff
@@ -469,6 +469,69 @@ Expected output if Trento web/wanda is ready and the database connection is setu
 {"ready":true}{"database":"pass"}
 ```
 
+## Integrations
+
+### OpenID Connect
+
+Trento integrates with an identity provider (IDP) that uses the OpenID Connect (OIDC) protocol to authenticate users accessing the Trento web console. Authorization for specific abilities/permissions is managed by Trento, which means that only basic user information is retrieved from the external IDP.
+
+
+#### User Roles and Authentication
+
+User authentication is entirely managed by the IDP, which is responsible for maintaining user accounts. 
+A user, who does not exist on the IDP, is unable to access the Trento web console.
+If the default admin user created during the installation process matches the authenticated user’s IDP username, that user is automatically granted `all:all` permissions within Trento.
+Create a new user on IDP and login to Trento to finalize the user creation on Trento.
+Set `all:users` permissions with the default admin user to the newly created admin user.
+
+All roles and permissions should be managed by the newly created admin user with the `all:users` permissions inside Trento, independent of the IDP.
+
+> **Note:** Multi-Factor Authentication cannot be enabled for the default admin user, keeping its password safe is essential
```

Comment:
You cannot enable this either way. MFA is not usable when 3rd party IDPs are used.
Authentication is managed by the IDP so trento cannot do anything about this.